clang-tools-extra/clang-tidy/NoLintDirectiveHandler.h

```diff
@@ -31,6 +31,8 @@ class NoLintDirectiveHandler {
 public:
   NoLintDirectiveHandler();
   ~NoLintDirectiveHandler();
+  NoLintDirectiveHandler(const NoLintDirectiveHandler &) = default;
+  NoLintDirectiveHandler &operator=(const NoLintDirectiveHandler &) = default;
```

Comment:
I don't think these are strictly needed, but I think it would be fine to explicitly `= delete` them.